ui/apps/platform/src/Containers/Clusters/ClusterEditFormLegacy.tsx

```diff
@@ -1,46 +1,41 @@
-import React, { ReactElement } from 'react';
+import React from 'react';
+import type { ReactElement } from 'react';
 import { Alert, Flex, FlexItem } from '@patternfly/react-core';
 
-import Loader from 'Components/Loader';
 import { labelClassName } from 'constants/form.constants';
-import { Cluster, ClusterManagerType } from 'types/cluster.proto';
-import { DecommissionedClusterRetentionInfo } from 'types/clusterService.proto';
+import type { Cluster, ClusterManagerType } from 'types/cluster.proto';
+import type { DecommissionedClusterRetentionInfo } from 'types/clusterService.proto';
 
-import ClusterSummary from './Components/ClusterSummary';
-import StaticConfigurationSection from './StaticConfigurationSection';
-import DynamicConfigurationSection from './DynamicConfigurationSection';
+import ClusterSummaryLegacy from './Components/ClusterSummaryLegacy';
+import StaticConfigurationSectionLegacy from './StaticConfigurationSectionLegacy';
+import DynamicConfigurationSectionLegacy from './DynamicConfigurationSectionLegacy';
 import ClusterLabelsTable from './ClusterLabelsTable';
 
-type ClusterEditFormProps = {
+type ClusterEditFormLegacyProps = {
     centralVersion: string;
     clusterRetentionInfo: DecommissionedClusterRetentionInfo;
     selectedCluster: Cluster;
     managerType: ClusterManagerType;
     handleChange: (any) => void;
     handleChangeLabels: (labels) => void;
-    isLoading: boolean;
 };
 
-function ClusterEditForm({
+function ClusterEditFormLegacy({
     centralVersion,
     clusterRetentionInfo,
     selectedCluster,
     managerType,
     handleChange,
     handleChangeLabels,
-    isLoading,
-}: ClusterEditFormProps): ReactElement {
-    if (isLoading) {
-        return <Loader />;
-    }
```

Comment:
Was it intentional to remove the loading part?

Reply:
Good eyes. I lifted loading state up to ClusterPage.tsx file:
* More consistent with usual loading, error, data rendering pattern.
* Replace `Loader` with PatternFly `Spinner` element.